apps/web/pages/event-types/[type].tsx

```diff
@@ -442,6 +445,10 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
     team ? `team/${team.slug}` : eventType.users[0].username
   }/${eventType.slug}`;
 
+  const placeholderHashedLink = `${process.env.NEXT_PUBLIC_WEBSITE_URL}/d/${
+    eventType.hashedLink ? eventType.hashedLink.link : "xxxxxxxxxxxxxxxxx"
+  }/${eventType.slug}`;
```

Comment:
The placeholder hashed link to show in the input field and for copying purposes, if the hashed link is already generated, else a fallback placeholder which won't be copied until saved/updated